models/estimate_fonll_nonprompt_fractions.py

```diff
@@ -0,0 +1,80 @@
+"""
+Utility script to estimate the "natural" non-prompt fraction from FONLL + PYTHIA8
+"""
+
+import sys
+import numpy as np
+from ROOT import TFile, TCanvas, TLegend #pylint: disable=import-error,no-name-in-module
+from ROOT import kRed, kOrange, kGreen #pylint: disable=import-error,no-name-in-module
+sys.path.append('..')
+from utils.StyleFormatter import SetGlobalStyle, SetObjectStyle #pylint: disable=wrong-import-position,import-error
+
+def main(): #pylint: disable=too-many-statements
+    SetGlobalStyle(padleftmargin=0.18, padbottommargin=0.14, titleoffsety=1.6, maxdigits=2, optstat=0)
+
+    inFileFONLL = TFile.Open('fonll/feeddown/DmesonLcPredictions_502TeV_y05_FFee_BRpythia8_SepContr_PDG2020.root')
+    hFONLLPromptD0 = inFileFONLL.Get('hD0Kpipred_max')
```

Comment:
Is it inted to choose max for prompt and central for non-prompt without considering the FONLL uncertainties?

Reply:
Yes, I was trying to get a "realistic" value. So upper edge for prompt and central for non-prompt. This of course could be refined and uncertainty added, not sure if we should do nonprompt_upper / prompt_upper and so on.